src/ClientData/CaptureDataTest.cpp

```diff
@@ -123,4 +132,49 @@ TEST_F(CaptureDataTest, UpdateScopeStatsIsCorrect) {
                    GetStats(kDurationsForSecondId, kSecondVariance));
 }
 
+TEST_F(CaptureDataTest, VarianceIsCorrectForLongDurations) {
+  for (TimerInfo timer : kTimerInfos) {
+    timer.set_end(timer.end() + kLargeInteger);
+    capture_data_.UpdateScopeStats(timer);
+  }
+
+  capture_data_.UpdateScopeStats(kTimerInfoWithInvalidScopeId);
+
+  ORBIT_LOG("VARIANCE %f", capture_data_.GetScopeStatsOrDefault(kFirstId).variance_ns());
+  EXPECT_NEAR(capture_data_.GetScopeStatsOrDefault(kFirstId).variance_ns(), kFirstVariance, 1.0);
+  EXPECT_NEAR(capture_data_.GetScopeStatsOrDefault(kSecondId).variance_ns(), kSecondVariance, 1.0);
+}
+
+TEST_F(CaptureDataTest, VarianceIsCorrectOnScimitarDataset) {
```

Comment:
Maybe add information about the number of entries in the data set? It could be interesting to put this in relation to a capture, like N events per second for an hour. Also I imagine we could parse the same data set multiple times to simulate longer captures if needed?

Reply:
> Maybe add information about the number of entries in the data set?

Add to the dataset or to the code? What for? 

> I imagine we could parse the same data set multiple times to simulate longer captures if needed

In general, yes. I would only add, we shouldn't exactly re-parse the dataset, just resample whatever was parsed initially.

Reply:
Also, the dataset (`scimitar_variance_and_durations.csv`) has absolutely no information about how long the capture lasted.

Reply:
I'm curious to know how many events the data set has. Let's say there are 60'000' events, this can be expressed as a minute long capture at 1000 events/second. That would not be representative of the size of capture we'd like to support and the test might not be strict enough. We are dealing with captures of millions of event, and the "running variance" error is proportional to the number of events, so I think we should try to measure the error introduced from a capture of millions of events. If the dataset is not big enough, I was suggesting re-ingesting the same data set multiple times, which will make the "running variance" error grow but should not have an effect on the actual variance.

Reply:
Great point. I have added the comments. Also, I have added a test that runs feeds the dataset to `CaptureData` 100 times. Of course, the error does accumulate, but the difference is negligible.

Reply:
Awesome, thanks for the change!